examples/react-app/src/index.tsx

```diff
@@ -131,7 +131,6 @@ ReactDOM.render(
       <PlmgSvgIcon icon={'home'} size={'6em'} />
       PlmgSvgIcon home
       <br />
-      <Dropdown />
```

Comment:
does not seem to have anything to do with this PR?

Reply:
This is true. I was being lazy. I'll make a separate one :)

Reply:
Reverted. Moved bug fix to [this PR](https://github.com/DuckyTeam/Plumage/pull/77)